### ISSUE TITLE
Button that input content

### ISSUE THREAD

**Issue by jan-dh** [verbb/vizy#248]
### Question

I would like to have the ability to let's say define a couple of buttons that create content instead of applying styles to content. For example:
- A button that inputs ✅ or ❌ to consistently populate comparison tables
- A button that output the correct unicode version of a quotation mark. As you can see [here](https://symbl.cc/en/collections/quotation-marks/)

A lot of the times these buttons would help to provide consistency across content input.

**Comment by engram-design** [comment]
You'll need to get familiar with the Vizy API for [adding buttons](https://verbb.io/craft-plugins/vizy/docs/template-guides/extending-vizy). Worth checking out the [user guides](https://verbb.io/craft-plugins/vizy/docs/user-guides). You'll also need to know a bit about [Tiptap](https://tiptap.dev/introduction), particularly [`insertContent`](https://tiptap.dev/api/commands/insert-content/#insertcontent).

**Comment by jan-dh** [comment]
Oh totally missed those. Looks super interesting! Thanks 🙏

**Comment by jan-dh** [comment]
Tried following along the user guide on "Creating a custom Node from scratch" & implemented the Emoji button like in the guide, however, this gives me this error:

<img width="900" alt="image" src="https://github.com/verbb/vizy/assets/5801781/637d9025-9f84-44a4-9004-b95fff2bc1cd">

Database gives me:
```
[{"type":"paragraph","attrs":[],"content":[{"type":"emoji","attrs":{"emoji":":fire:"},"text":""}]}]
```

**Comment by engram-design** [comment]
So that'll just be related to some refactoring we're doing. Fixed for the next release. To get this early, run `composer require verbb/vizy:"dev-craft-4 as 2.1.8"`

**Comment by engram-design** [comment]
Fixed in [2.1.9](https://github.com/verbb/vizy/releases/tag/2.1.9)
